marshmallow_jsonapi/schema.py

```diff
@@ -275,7 +275,7 @@ def format_item(self, item):
                 if 'meta' not in ret:
                     ret['meta'] = self.dict_class()
                 ret['meta'].update(value)
-            elif isinstance(self.fields[attribute], BaseRelationship):
+            elif isinstance(self.fields[attribute], BaseRelationship) and value:
```

Comment:
Is this change necessary? It seems possibly incorrect. If `value` is `None` then the field will be added to `attributes` rather than `relationships`.